lib/dash/segment_template.js

```diff
@@ -158,7 +158,8 @@ shaka.dash.SegmentTemplate = class {
       if (info.timeline &&
           context.adaptationSet.contentType !== 'image' &&
           context.adaptationSet.contentType !== 'text') {
-        const timeline = info.timeline;
+        const tsi = /** @type {!TimelineSegmentIndex} */(segmentIndex);
+        const timeline = tsi.getTimeline();
```

Comment:
This needs a comment to explain that tsi.getTimeline is different from info.timeline.

Reply:
Done!